lib/main.js

```diff
@@ -293,14 +293,16 @@ reporterPanel.sendAddonReports = function() {
                             by=220;
                         if (by<110)
                             by=110;
-                        let panelSize = 240+(by-110);
+                        let panelSize = 250+(by-110);
```

Comment:
You increase the panel `height` by 20px, but this is increased by 10px only. Is this intended? Did you test the layout on all three platforms? (We've run into troubles with this before.)